test/helpers/kubectl.go

```diff
@@ -2918,43 +2918,23 @@ func (kub *Kubectl) getPodRevisions() (map[string]int, error) {
 }
 
 func (kub *Kubectl) waitNextPolicyRevisions(podRevisions map[string]int, mustHavePolicy bool, timeout time.Duration) error {
-	npFilter := fmt.Sprintf(
-		`{range .items[*]}{"%s="}{.metadata.name}{" %s="}{.metadata.namespace}{"\n"}{end}`,
-		KubectlPolicyNameLabel, KubectlPolicyNameSpaceLabel)
-
-	knpBody := func() bool {
-		knp := kub.ExecShort(fmt.Sprintf("%s get --all-namespaces netpol -o jsonpath='%s'",
-			KubectlCmd, npFilter))
-		result := knp.ByLines()
-		if len(result) == 0 {
-			return true
-		}
-
-		for _, item := range result {
-			for ciliumPod, revision := range podRevisions {
-				if mustHavePolicy {
```

Comment:
It looks like `mustHavePolicy` is no longer used after this change. Can we remove it from `waitNextPolicyRevisions` arguments?

Reply:
Done :-)